frontend/src/components/GoalCards/__tests__/GoalCards.js

```diff
@@ -286,6 +286,8 @@ const renderTable = ({ goals, goalsCount, allGoalIds = null }, user, hasActiveGr
             }}
             setGoals={setGoals}
             allGoalIds={allGoalIds || goals.map((g) => g.id)}
+            shouldDisplayMergeSuccess={false}
+            dismissMergeSuccess={jest.fn()}
```

Comment:
Should we be testing these here or are they tested in another location?

Reply:
They are tested in another location